python/py-sacrebleu/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        mjpost sacrebleu 1.5.1 v
+revision            0
+name                py-${github.project}
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Reference BLEU implementation that auto-downloads \
+                    test sets and reports a version string to \
+                    facilitate cross-lab comparisons
```

Comment:
this is a bit long for a "short" description...

Reply:
Thanks, took the "shorter" one from PyPi…